pkg/kcp/provider/gcp/redisinstance/new.go

```diff
@@ -40,15 +40,15 @@ func New(stateFactory StateFactory) composed.Action {
 				composed.ComposeActions(
 					"redisInstance-create",
 					createRedis,
-					updateStatusId,
+					PatchStatusId,
```

Comment:
Could you just use the lower case for the first letter?

Reply:
I reverted function name changes